src/indicators/flooding/floodscan/floodscan.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import numpy as np
+import geopandas as gpd
+from rasterstats import zonal_stats
+import xarray as xr
+import rioxarray
+import logging
+
+
+from pathlib import Path
+import sys
+import os
+
+path_mod = f"{Path(os.path.dirname(os.path.realpath(__file__))).parents[1]}/"
+sys.path.append(path_mod)
+from src.indicators.drought.config import Config
+
+config=Config()
+
+DATA_DIR = Path(os.environ["AA_DATA_DIR"])
+DATA_PRIVATE_DIR = Path(os.environ["AA_DATA_PRIVATE_DIR"])
+RAW_DATA_DIR = "raw"
+SHAPEFILE_DIR = config.SHAPEFILE_DIR
+PROCESSED_DATA_DIR = "processed"
+FLOODSCAN_DIR = Path("floodscan")
+FLOODSCAN_FILENAME = "floodscan_africa_sfed_area_300s_19980112_20201231_v05r01.nc"
+
+DEFAULT_ADMIN_LEVEL = 1
+
+logger = logging.getLogger(__name__)
+
+
+class Floodscan():
+    """
+    Create an instance of a Floodscan object, from which you can process the raw data and read the data.
+    """
+    def read_raw_dataset(self):
+        filepath=self._get_raw_filepath()
+        #would be better to do with load_dataset, but since dataset is huge this takes up too much memory..
+        return xr.open_dataset(filepath)
+
+    def process(self,
+                country_name: str,
+                adm_level: int = DEFAULT_ADMIN_LEVEL,
+                ):
+        """
+        Load data, call function to compute statistics per admin, and save the results to a csv
+        Args:
+            country_name: name of the country of interest
+            adm_level: admin level to compute the statistics on
+        """
+        config=Config()
+        parameters = config.parameters(country_name)
+        adm_boundaries_path = os.path.join(DATA_DIR,RAW_DATA_DIR,country_name,config.SHAPEFILE_DIR,parameters[f"path_admin{adm_level}_shp"])
+        ds=self.read_raw_dataset()
+        #get the affine transformation of the dataset. looks complicated, but haven't found better way to do it
+        coords_transform=ds.rio.set_spatial_dims(x_dim="lon",y_dim="lat").rio.write_crs("EPSG:4326").rio.transform()
```

Comment:
Yes you are totally right. What it is doing here is not to change the CRS, but to add a crs, since the file didn't have one. So I think in that use case it is fine to have it hardcoded instead of as a variable, or would you say differently?
However, I assumed the floodscan data to be in WGS84. But actually don't know if that is correct..